manim/mobject/value_tracker.py

```diff
@@ -71,7 +74,7 @@ def __init__(self, value=0, **kwargs):
         self.set_value(value)
 
     def get_value(self) -> float:
-        """Get the current value of the ValueTracker. This value changes continuously when :attr:`animate` for the ValueTracker is called."""
+        """Get the current value of this value tracker."""
```

Comment:
```suggestion
        """Get the current value of this ValueTracker."""
```

Reply:
I just saw that it is all the time this value tracker, so we can also keep it like that.